lib/ffi/aspell/aspell.rb

```diff
@@ -13,8 +16,21 @@ module FFI
   # For more information see {FFI::Aspell::Speller}.
   #
   module Aspell
-    extend   FFI::Library
-    ffi_lib ['aspell', 'libaspell.so.15']
+    extend FFI::Library
+
+    begin
+      stdout, stderr, status = ::Open3.capture3("brew", "--prefix")
+      homebrew_path  = if status.success?
+                        "#{stdout.chomp}/lib"
+                      else
+                        '/usr/local/homebrew/lib'
+                      end
+    rescue
+      # Homebrew doesn't exist
+    end
+
+    ffi_lib ['aspell', 'libaspell.so.15'] if ::RbConfig::CONFIG['host_os'] =~ /linux/
+    ffi_lib ["#{homebrew_path}/libaspell.dylib"] if ::RbConfig::CONFIG['host_os'] =~ /darwin/
```

Comment:
@petems I really don't like this kind of logic. The way this works we'd have to add similar code for every package manager, and perhaps different configurations. This code will also run and fail on non OS X systems.

Long story short, I want a transparent solution that doesn't add Homebrew/Fink/Macports/etc specific code to this Gem.